content/cloud-docs/api-docs/index.mdx

```diff
@@ -349,7 +349,7 @@ HashiCorp maintains [go-tfe](https://github.com/hashicorp/go-tfe), a Go client f
 
 Additionally, the community of Terraform Cloud users and vendors have built client libraries in other languages. These client libraries and tools are not tested nor officially maintained by HashiCorp, but are listed below in order to help users find them easily.
 
-If you have built a client library and would like to add it to this community list, please [contribute](https://github.com/hashicorp/terraform-website) to [this page](https://github.com/hashicorp/terraform-website/blob/master/content/source/docs/cloud/api/index.html.md).
+If you have built a client library and would like to add it to this community list, please [contribute](https://github.com/hashicorp/terraform-website#contributions-welcome) to [this page](https://github.com/hashicorp/terraform-website/blob/87f49d5caa6d281982acff996c0ad5edc50f7653/content/cloud-docs/api-docs/index.mdx#client-libraries-and-tools).
```

Comment:
updating this to `master` branch

```suggestion
If you have built a client library and would like to add it to this community list, please [contribute](https://github.com/hashicorp/terraform-website#contributions-welcome) to [this page](https://github.com/hashicorp/terraform-website/blob/master/content/cloud-docs/api-docs/index.mdx#client-libraries-and-tools).
```

Reply:
this is out of scope of this PR but want to track this down. can we update terraform-website from `master` -> `main`? the core terraform repo uses `main`

Reply:
Let's talk to web dev about this!